backend/init_app.py

```diff
@@ -11,6 +11,7 @@
 from resources.reviews import Reviews
 
 from db import db, init_db
+from resources.verify_email import VerifyEmail
```

Comment:
Tonteria perque sigui consisten els imports
```suggestion
from resources.reviews import Reviews
from resources.verify_email import VerifyEmail

from db import db, init_db
```